package.json

```diff
@@ -1,6 +1,13 @@
 {
   "name": "@snyk/kubernetes-monitor",
-  "description": "Snyk kubernetes Monitioring Container",
+  "private": true,
+  "description": "Snyk Kubernetes Monitoring Container",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/snyk/kubernetes-monitor.git"
+  },
+  "license": "private",
```

Comment:
So, not entirely related to this PR specifically, but the repo license is Apache 2.0, should we change this?

Reply:
Good spot, probably yes :)